nilearn/regions/tests/test_signal_extraction.py

```diff
@@ -256,29 +307,6 @@ def test_signal_extraction_with_maps():
     img_r = signal_extraction.signals_to_img_maps(signals, maps_img)
     np.testing.assert_almost_equal(get_data(img_r), get_data(img))
 
-    # Test input validation
-    data_img = nibabel.Nifti1Image(np.zeros((2, 3, 4, 5)), np.eye(4))
-
-    good_maps_img = nibabel.Nifti1Image(np.zeros((2, 3, 4, 7)), np.eye(4))
-    bad_maps1_img = nibabel.Nifti1Image(np.zeros((2, 3, 5, 7)), np.eye(4))
-    bad_maps2_img = nibabel.Nifti1Image(np.zeros((2, 3, 4, 7)), 2 * np.eye(4))
-
-    good_mask_img = nibabel.Nifti1Image(np.zeros((2, 3, 4)), np.eye(4))
-    bad_mask1_img = nibabel.Nifti1Image(np.zeros((2, 3, 5)), np.eye(4))
-    bad_mask2_img = nibabel.Nifti1Image(np.zeros((2, 3, 4)), 2 * np.eye(4))
-    pytest.raises(ValueError, signal_extraction.img_to_signals_maps, data_img,
-                  bad_maps1_img)
-    pytest.raises(ValueError, signal_extraction.img_to_signals_maps, data_img,
-                  bad_maps2_img)
-    pytest.raises(ValueError, signal_extraction.img_to_signals_maps, data_img,
-                  bad_maps1_img, mask_img=good_mask_img)
-    pytest.raises(ValueError, signal_extraction.img_to_signals_maps, data_img,
-                  bad_maps2_img, mask_img=good_mask_img)
-    pytest.raises(ValueError, signal_extraction.img_to_signals_maps, data_img,
-                  good_maps_img, mask_img=bad_mask1_img)
-    pytest.raises(ValueError, signal_extraction.img_to_signals_maps, data_img,
```

Comment:
Same comment here

Reply:
If these are retained, they are a good candidate for `pytest.mark.parametrize`.